test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/ArrayDequeTest.scala

```diff
@@ -16,6 +16,7 @@ import java.util
 
 import org.junit.Test
 import org.junit.Assert._
+import org.scalajs.testsuite.utils.AssertThrows.assertThrows
```

Comment:
I have a feeling there will be complaints about the ordering of imports 😅

Reply:
Ideally, it would come last. ;) Things in the "current project" always come last.